python/ray/tune/suggest/bayesopt.py

```diff
@@ -61,10 +64,11 @@ class BayesOptSearch(Searcher):
 
     def __init__(self,
                  space,
-                 metric="episode_reward_mean",
+                 metric,
```

Comment:
can we leave this in? this may break existing codes

Reply:
I thought that since there was no true default metric it was misleading to live that in. Am I mistaken?

Reply:
hmm, I guess that's a good point